tests/components/CopyPaste/CopyPaste.spec.ts

```diff
@@ -62,6 +70,11 @@ describe('CopyToClipboard ResultAction', () => {
                 testComponent.element.click();
                 expect(execCommandStub.called).toBeTrue();
             });
+
+            it('should change the tooltip text for "Copied!"', async () => {
```

Comment:
You should add tests to validate tooltip text when using custom `options.tooltip` and `options.successTooltip`.